dll/cpl/access/lang/pt-PT.rc

```diff
@@ -262,16 +262,16 @@ END
 
 IDD_SERIALKEYSOPTIONS DIALOGEX 0, 0, 246, 228
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Configurações de dispositivos seriais"
+CAPTION "Definições para ""Teclas série"""
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Teclas seriais", -1, 6, 11, 234, 92
-    LTEXT "Escolha a porta para a ligação de um dispositivo de entrada alternativo.",
+    GROUPBOX "Teclas série", -1, 6, 11, 234, 92
+    LTEXT "Seleccione a porta para ligação de um dispositivo de entrada alternativo.",
           -1, 12, 20, 222, 20
-    LTEXT "&Porta serial:", -1, 12, 34, 222, 20
+    LTEXT "&Porta série:", -1, 12, 34, 222, 20
     COMBOBOX IDC_SERIAL_PORT_COMBO, 12, 45, 222, 56,
              CBS_DROPDOWNLIST | WS_VSCROLL | WS_VISIBLE | WS_TABSTOP
-    LTEXT "&Taxa de transm.:", -1, 12, 65, 222, 20
+    LTEXT "&Velocidade de transmissão:", -1, 12, 65, 222, 20
```

Comment:
Can you check whether the text zone is large enough?

Reply:
@HBelusca  The button that gives access to this dialog is disabled, however I believe there is free space for all the text, as it has the entire width of the window

Reply:
![KSerialAfter](https://user-images.githubusercontent.com/49964946/82228000-6b836d80-9920-11ea-91ce-28b6d8885ce4.png)
Can we expand the window a little bit here?